### PR TITLE
HTCONDOR-2253 cleanup grid resource

### PR DESCRIPTION
**Pull request by JaimeFrey**

### CODE REVIEW

File: supported/osg-htc/osg-hosted-ce/templates/configmap.yaml [hunk 1]
```diff
@@ -138,17 +138,14 @@ data:
...
-        gr = $(gr) --rgahp-script batch_gahp
-        {{ end }}
-        {{ if not .Values.RemoteCluster.SSHBatchMode }}
-        gr = $(gr) --rgahp-nobatchmode
-        {{ end }}
-        {{ if not .Values.RemoteCluster.LoginShell }}
-        gr = $(gr) --rgahp-nologin
-        {{ end }}
+        hosted_grid_resource = batch {{ .Values.RemoteCluster.Batch }} $(MY.Owner)@{{ .Values.RemoteCluster.LoginHost }} --rgahp-glite ~/{{ .Values.RemoteCluster.BoscoDir | default "~/bosco"}}/glite
+        {{- if .Values.BoscoOverrides.TarballURL | default "" | contains "bosco-1.3" }} --rgahp-script batch_gahp{{ end }}
```
Comment (brianhlin) [inline supported/osg-htc/osg-hosted-ce/templates/configmap.yaml]:
```suggestion
```

I think we can actually drop this one. There don't appear to be any CEs left that are using such an old Bosco tarball

**Review by brianhlin — APPROVED** [review]
LGTM, one suggestion on removing some old hacks that we needed I think around the time we merged the blahp into HTCondor

File: supported/osg-htc/osg-hosted-ce/templates/configmap.yaml (hunk 1 above)
Comment (JaimeFrey) [reply supported/osg-htc/osg-hosted-ce/templates/configmap.yaml]: I see one CE with a live TarballURL value: tacc-stampede2-ce2

Comment (JaimeFrey) [reply supported/osg-htc/osg-hosted-ce/templates/configmap.yaml]: Ah, the change here is about a specific ancient bosco tarball.

**Review by brianhlin — APPROVED** [review]
LGTM!

**Pull request merged**